packages/shared/src/components/post/SquadPostAuthor.tsx

```diff
@@ -107,6 +107,14 @@ function SquadPostAuthor({
             </TruncateText>
             {!!date && <Separator />}
             {!!date && <DateFormat date={date} type={TimeFormatType.Post} />}
+            {author?.topReader && (
+              <>
+                <Separator />
+                <TruncateText>
+                  Top reader in {author.topReader?.keyword?.flags?.title}
+                </TruncateText>
```

Comment:
Should use <TopReaderIn topReader={topReader} /> right?
```suggestion
                <TopReaderIn topReader={author.topReader} />
```